### ISSUE TITLE
Full Review of M2

### ISSUE THREAD

**Issue by ibogicevic** [dhbw-stginf16a/Rebleyama#86]
At the end of M2, we do a technical review about the current status of the implementation. We need one moderator, one actuator, and four reviewers. If you want to join the review, please assign yourself to this issuse and post your role as a comment. First come first serve. When the reviewers prepare to the review meeting, they should have a look both at the code and the running game. Especially they should look for:
- structure of the implementation
- general code quality
- structure of the code
- existence and quality of code comments
- any open sonarlint-issue
- non-working use cases
- missing use cases (not planned so far)
- errors and strage behaviour of the program
Please don't start reviewing before we haven't defined the exact revision to review.

**Comment by ibogicevic** [comment]
The issues found in the review meeting should be protocolled as single github-issues (issue by issue) and all be marked with the label "fromReview". All other review information (time of review meeting, preparation time of reviewers, general impression)

**Comment by ibogicevic** [comment]
Date of Review: May 16th 2018

**Comment by ibogicevic** [comment]
Postponed to MX.
